components/src/core/Drawer/DrawerNavItem.tsx

```diff
@@ -89,6 +89,11 @@ export type NestedNavItem = NestedDrawerNavItemProps;
 const calcNestedPadding = (theme: Theme, depth: number): number =>
     theme.spacing(depth ? (depth - 1) * 4 : 0) + theme.spacing(2);
 
+const getChevronColor = (props: DrawerNavItemProps, theme: Theme): string => {
+    const { chevronColor } = props;
+    return chevronColor ? chevronColor : theme.palette.text.secondary;
+};
+
```

Comment:
I see now why we had to do this. I thought we had an issue logged for not being able to combine chevron and rightComponent on the InfoListItem, but it looks like not, so I'll add a new issue for that. What you have here looks fine until we refactor the ILI.